doc/guide/cockpit-channel.xml

```diff
@@ -253,30 +253,32 @@ cockpit.transport.close([problem])
   <refsection id="cockpit-transport-filter">
     <title>cockpit.transport.filter()</title>
 <programlisting>
-cockpit.transport.filter(function(message, channel, control) { ... })
+cockpit.transport.filter(function(message, channelid) { ... }, [out])
```

Comment:
This is a incompatible change, no?  Old code might still expect the `control` argument to be passed.  Do we need to worry about that?